Source/WebCore/Modules/mediasource/MediaSource.cpp

```diff
@@ -1702,6 +1730,11 @@ Ref<SourceBufferList> MediaSource::activeSourceBuffers() const
     return m_activeSourceBuffers;
 }
 
+RefPtr<MediaSourcePrivate> MediaSource::protectedPrivate() const
```

Comment:
Should we inline it?

Reply:
it gives compilation error as MediaSourcePrivate is only fast forwarded; I would need to pull the entire MediaSourcePrivate.h header